File: Lecture3/Metropolis.java
import java.util.Scanner;

public class Metropolis {
  public static void main(String[] args) {
    boolean isCapital, isMetropolis;
    int citizen;
    int bourgeois;

    Scanner sc = new Scanner(System.in);
    System.out.print("Is the city capital? (capital:1 non-capital:0) ");
    isCapital = (sc.nextInt() == 1);
    System.out.print("Population? (in thousands) ");
    citizen = sc.nextInt();
    System.out.print("Bourgeois? (in thousands) ");
    bourgeois = sc.nextInt();

    isMetropolis = (isCapital && citizen >= 1000) || (bourgeois >= 500);

    System.out.println("Metropolis: " + isMetropolis);
  }
}

File: Lecture3/Metropolis.c
#include <stdio.h>

int main(void) {
    int isCapital = 0, isMetropolis = 0;
    int citizen;
    int bourgeois;

    printf("Is the city capital? (capital:1 non-capital:0) ");
    scanf("%d", &isCapital);
    printf("Population? (in thousands) ");
    scanf("%d", &citizen);
    printf("Bourgeois? (in thousands) ");
    scanf("%d", &bourgeois);

    isMetropolis = (isCapital && citizen >= 1000) || (bourgeois >= 500);

    printf("Metropolis: %s\n", isMetropolis ? "true" : "false");

    return 0;
}
